extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -99,8 +99,17 @@ oppia.directive('oppiaInteractiveImageClickInput', [
               return 'inline';
             }
           };
+          $scope.getDotDisplay = function() {
+            if (ExplorationContextService.getEditorTabContext() === 'editor') {
```

Comment:
Can you use the constant EDITOR_TAB_CONTEXT.EDITOR instead of 'editor'?

Reply:
Done